lib/internal/test_runner/test.js

```diff
@@ -1002,6 +1004,22 @@ class Test extends AsyncResource {
 
       if (coverage) {
         reporter.coverage(nesting, loc, coverage);
+
+        const coverages = [
+          [coverage.totals.coveredLinePercent, this.config.minimumLineCoveragePercent, 'line'],
```

Comment:
Using an object instead on an array would make it much more readable

Reply:
+1; I believe nyc/istanbul's `.nycrc` also uses an object with named values here